thirdparty/assimp/.gitignore

```diff
@@ -0,0 +1,104 @@
+.idea
```

Comment:
I don't think we need assimp's `.gitignore`.

Reply:
Fixed!